_posts/2017-08-08-CompetitiveCoding.md

```diff
@@ -0,0 +1,123 @@
+
+**Codeforces**-
+Codeforces is a Russian website dedicated to competitive programming.
+
+If you are a beginner, then it's better to go for SPOJ as it has problems classified on basis of specific data structure/algorithm involved. Never rush into participating in contests because it will be of no use practically till you carry a good set of tools behind you to tackle the problems. 
+*Beware*!! You must switch to codechef or codeforces only when you are ready to enter the competitive phase. The problems you'll see there might seem to be a little more advanced than required at first.
+
+**Special Mention- GeekforGeeks**
+ is something you're bound to come across if you are to stay in the world of competitive programming for a while. It is an excellent platform that contain articles on almost every topic (Algos, Data structures, Specific Problems) that can help you grasp a concept from the scratch. The articles not only include the brief idea but also the implementation details.
+[http://www.geeksforgeeks.org/]
+
+
+ 
+### Some prestigious competitions to target (to inspire you or to give you a sense of a goal)
+
+**ACM-ICPC**
+ACM International Collegiate Programming Contest (abbreviated as ACM-ICPC or ICPC) is an annual multi-tiered competitive programming competition among the universities of the world. The contest is sponsored by IBM. ICPC contests are team competitions. Current rules stipulate that each team consist of three students. Participants must be university students, who have had less than five years of university education before the contest. Students who have previously competed in two World Finals or five regional competitions are ineligible to compete again.
+During each contest, the teams of three are given 5 hours to solve between 8 and 15 programming problems (with 8 typical for regionals and 12 for finals). They must submit solutions as programs in C, C++, Java or Python (although it is not guaranteed every problem is solvable in Python). Programs are then run on test data. If a program fails to give a correct answer, the team is notified and can submit another program.
+
+**Facebook Hacker-Cup**
+Facebook Hacker Cup is an international programming competition hosted and administered by Facebook. The competition began in 2011 as a means to identify top engineering talent for potential employment at Facebook.The competition consists of a set of algorithmic problems which must be solved in a fixed amount of time. Competitors may use any programming language and development environment to write their solutions.
+
+**Codechef-Snackdown**
+CodeChef SnackDown is an annual multi-round global programming tournament conducted by CodeChef. It aims to pit the best programming minds from across the globe against each other for the ultimate title of “SnackDown Champions”. It was conceived on the model of ICPC in its first edition in the year 2010 and catered to university students in India.
+ 
```

Comment:
I avoided mentioning google codejam as it is not as targetable as the others mentioned in the post for many reasons....but since you brought it up...i have added it..